plugins/search-react/src/components/SearchBar/SearchBar.tsx

```diff
@@ -139,22 +133,30 @@ export const SearchBarBase: ForwardRefExoticComponent<SearchBarBaseProps> =
 
       const endAdornment = (
         <InputAdornment position="end">
-          <IconButton aria-label="Clear" size="small" onClick={handleClear}>
-            <ClearButton />
-          </IconButton>
+          <Button aria-label="Clear" size="small" onClick={handleClear}>
```

Comment:
When I'm navigating to this button with keyboard now and press "Enter" it closes the modal instead of clearing the results which wasn't the case with the old clear button, can you look into how that can be fixed before we merge this? Apart from that its good to go!

Reply:
Fixed